### DISCUSSION TITLE
控制中心评论跳转问题

### DISCUSSION THREAD

**Discussion by inkss** [ArtalkJS/Artalk#110] [Q&A] [answered]
我又来提BUG了，(lll￢ω￢)

**前端版本：** `v2.2.3`

**现象：**

You Know，我全程使用相对链接的，在控制中心点击评论时，跳转的链接成了当前链接的完整地址+评论地址，这里应该只是域名部分吧：`location.origin`。

**Comment by qwqcode** [comment]
诶，我记得我修过这个 bug，我看看

**Comment by qwqcode** [comment, marked as answer]
已修复，待会发个新版 https://github.com/ArtalkJS/Artalk/commit/7577374e4b5598ce4bf2895c2cfc8145c9672f62
